### PR TITLE
fix(theme): change source icon size

### PR DESCRIPTION
**Pull request by ikechan8370**
The icon has been shrunk so that it can be displayed in full.
before:

after:

### CODE REVIEW

**Comment by Mister-Hope** [comment]
BTW, if you have idea replacing currect icon with a better one, that's fine either.

**Comment by ikechan8370** [comment]
> BTW, if you have idea replacing currect icon with a better one, that's fine either.
> 
> It does do not look well.

Apart from the common git icons like github, I really think the code icon is the most accurate one to represents the meaning of "source code repository". Maybe we can try another style? Just like these from iconfont:

11681649934147_ pic11691649934147_ pic

Or the "repo" icon:
11671649934147_ pic
But it doesn't remind me of "source code repository" at first sight when I see it.
BTW, I think maybe the custom repository icon option is really needed.

**Comment by Mister-Hope** [comment]
Maybe a "fork" icon?

**Comment by Mister-Hope** [comment]
I just though you would have some ideas since you want to customize it.

**Comment by ikechan8370** [comment]
11711649935941_ pic
Fork is also a good idea. Does it look good?
And I just meant to use a Gitlab icon when the repo url is a domain of our private gitlab server instead of "gitlab.com" domain. And in some cases, if someone has a repository on other git server such as gitea, he can also use a gitea logo as a icon. If so, I think it will be quite rare to use this source icon.

**Comment by Mister-Hope** [comment]
> 11711649935941_ pic Fork is also a good idea. Does it look good? And I just meant to use a Gitlab icon when the repo url is a domain of our private gitlab server instead of "gitlab.com" domain. And in some cases, if someone has a repository on other git server such as gitea, he can also use a gitea logo as a icon. If so, I think it will be quite rare to use this source icon.

Yes this is better, please update your pr

File: packages/theme/src/client/module/navbar/components/icons/repoIcon.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ BitbucketIcon.displayName = "BitbucketIcon";
 export const SourceIcon: FunctionalComponent = () =>
```
Comment (Mister-Hope) [inline packages/theme/src/client/module/navbar/components/icons/repoIcon.ts:42]: Here? Also I think just keeping the original icon name should be fine.

**Review by Mister-Hope — CHANGES_REQUESTED** [review]

File: packages/theme/src/client/module/navbar/components/icons/repoIcon.ts (hunk 1 above)
Comment (ikechan8370) [reply packages/theme/src/client/module/navbar/components/icons/repoIcon.ts:42]: Ok, I'll fix it soon. Just wait a moment, I have some problems with my IDE. Webstorm frequently crashed after updating to 2022.1. i will roll back to 2021.

Comment (ikechan8370) [reply packages/theme/src/client/module/navbar/components/icons/repoIcon.ts:42]: Do you mean to just replace svg content? Or to replace the "SourceIcon" with a new "ForkIcon".

**Pull request merged**